damus/Features/Timeline/Models/HomeModel.swift

```diff
@@ -543,6 +545,17 @@ class HomeModel: ContactsDelegate, ObservableObject {
                 }
             }
         }
+        // Due to subscription volume limits in ndb and in relays, some important events may get clipped in the `generalHandlerTask` above.
+        // This could lead to issues (e.g. The app overriding a mutelist because it does not have it)
+        // Therefore, we have this ndb-only stream for some low volume important items.
+        // The reason we do not separate into two complete streams is because
+        // we need to keep total relay subscription down to avoid relay subscription limits.
+        self.ndbOnlyHandlerTask?.cancel()
+        self.ndbOnlyHandlerTask = Task {
+            for await eventLender in damus_state.nostrNetwork.reader.streamIndefinitely(filters: low_volume_important_filters, streamMode: .ndbOnly) {
+                await eventLender.justUseACopy({ await process_event(ev: $0, context: .other) })
```

Comment:
is there a reason process_event has await on it? this function call is instantaneous and threadsafe on its own